src/drive/web/modules/views/Modal/ShareView.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { useNavigate } from 'react-router-dom'
+
+import { ShareModal } from 'cozy-sharing'
+
+import { useDisplayedFolder } from 'drive/hooks'
+
+const ShareView = () => {
+  const displayedFolder = useDisplayedFolder()
+  const navigate = useNavigate()
+
+  if (displayedFolder) {
+    const onClose = () => {
+      // navigate(`/folder/${displayedFolder.id}`)
+      navigate(-1)
```

Comment:
Don't use -1, we have issue with this on the flagship app. You can use `..` or the path you want, but not -1 🙏

Reply:
It's replaced